Update bms_ranges_test.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+import bms_ranges_monitor
+
+class test_battery_current_ranges(unittest.TestCase):
+  def test_handled_failing_current_ranges(self):
+    self.assertTrue(bms_ranges_monitor.current_ranges([]) == "Invalid Input")
+    
+  def test_passing_current_ranges(self):
+    self.assertTrue(bms_ranges_monitor.current_ranges([3, 3, 4, 7 ,10, 11, 12, 13]) == "Valid Input") 
```

Comment:
These tests check valid and invalid input, but give no clue about the ranges or counts.
Write tests for the list_tuple.